module/config.mjs

```diff
@@ -113,6 +113,12 @@ ED4E.globalBonuses = {
 	allRangedDamage: {
 		label: 			"ED.Actor.GlobalBonus.allRangedDamage"
 	},
+	allRecoveryEffects: {
+		label: 			"ED.Actor.GlobalBonus.allRecoveryEffects"
+	},
+	allKnockdownEffects: {
```

Comment:
Are there multiple knockdown effects?

Reply:
you can have threads providing bonus to tests agains knockdown from group pattern, items and what not

Reply:
ja es ist **kein globaler Bonus** es ist ein **individueller Bonus** es ist kein Fähnchen sondern in der Auswahl der Boni

Reply:
und bitte auch einbauen Herr Mohrmann nich tnur anllegen